testsuite/MDAnalysisTests/topology/test_top.py

```diff
@@ -154,8 +155,16 @@ class TestPRMParser(TOPBase):
     ref_filename = PRM
     # Does not contain an ATOMIC_NUMBER record, so no elements
     expected_attrs = [
-        "names", "types", "type_indices", "charges", "masses", "resnames",
-        "bonds", "angles", "dihedrals", "impropers"
+        "names",
+        "types",
+        "type_indices",
+        "charges",
+        "masses",
+        "resnames",
+        "bonds",
+        "angles",
+        "dihedrals",
+        "impropers",
```

Comment:
What changed here? If it's just a reformatting thing, could you revert this? It's a lot of extra noise on the diff that will make git blaming difficult if it ever needs to happen.

Reply:
Well, I can revert that if you want, but Darker asks for it. If the idea was to go towards flake8/black compliance, then stuff like this is gonna happen.

Reply:
As per the text of the darker bot, if it's not a flake8 failure then it's ok to ignore.

The flake8 failures are there but don't need addressing because we don't enforce changes on datafiles.

Reply:
sorry, I misread that. I thought it was my new `TestPRMChainidParser` class, not the old `TestPRMParser`. It's fixed now.